src/cljs/rems/administration/resource.cljs

```diff
@@ -0,0 +1,103 @@
+    ::fetch-resource [resource-id]}))
+
+(defn- fetch-resource [resource-id]
+  (fetch (str "/api/resources/" resource-id)
+         {:handler #(rf/dispatch [::fetch-resource-result %])}))
+
+(rf/reg-fx ::fetch-resource (fn [[resource-id]] (fetch-resource resource-id)))
+
+(rf/reg-event-db
+ ::fetch-resource-result
+ (fn [db [_ resource]]
+   (-> db
+       (assoc ::resource resource)
+       (dissoc ::loading?))))
+
+(rf/reg-sub ::resource (fn [db _] (::resource db)))
+(rf/reg-sub ::loading? (fn [db _] (::loading? db)))
+
+(defn- back-button []
+  [:button.btn.btn-secondary
+   {:on-click #(dispatch! "/#/administration/resources")}
+   (text :t.administration/back)])
+
+(defn- to-create-resource []
+  [:a.btn.btn-primary
+   {:href "/#/administration/create-resource"}
+   (text :t.administration/create-resource)])
+
+(defn inline-info-field [text value]
+  [info-field text value {:inline? true}])
+
+(defn license-view [license language]
+  (into [:div.form-item
+         [:h4 (text-format :t.administration/license-field (get-in license [:localizations language :title]))]
+         [inline-info-field (text :t.administration/title) (:title license)]]
+        (concat (for [[langcode localization] (:localizations license)]
+                  [inline-info-field (str (text :t.administration/title)
+                                          " "
+                                          (str/upper-case (name langcode))) (:title localization)])
+                [[inline-info-field (text :t.administration/type) (:licensetype license)]
+                 (case (:licensetype license)
+                   "link" [inline-info-field (text :t.create-license/external-link) [:a {:target :_blank :href (:textcontent license)} (:textcontent license) " " [external-link]]]
+                   "text" [inline-info-field (text :t.create-license/license-text) (:textcontent license)]
+                   nil)]
+                (when (= "link" (:licensetype license))
+                  (for [[langcode localization] (:localizations license)]
+                    [inline-info-field (str (text :t.create-license/external-link)
+                                            " "
+                                            (str/upper-case (name langcode))) [:a {:target :_blank :href (:textcontent localization)} (:textcontent localization) " " [external-link]]]))
+                [[inline-info-field (text :t.administration/start) (localize-time (:start license))]
+                 [inline-info-field (text :t.administration/end) (localize-time (:end license))]])
+        ))
+
+(defn licenses-view [licenses language]
+  [collapsible/component
+   {:id "licenses"
+    :title (text :t.administration/licenses)
+    :top-less-button? true
+    :collapse (if (seq licenses)
+                (into [:div]
+                      (for [license licenses]
+                        [license-view license language]))
+                [:p (text :t.administration/no-licenses)])}])
```

Comment:
Instead of showing the "no licenses" text only after pressing "show more", it would be more user friendly to show the "no licenses" text right away and remove the "show more" link in that case.

Reply:
And in case there are licenses, they could be shown right away. At least the license title and link (in current user language) could be shown without the need to press "show more".

Reply:
Already implemented in another PR